cg/store/api/find_business_data.py

```diff
@@ -3,18 +3,35 @@
 import logging
 from typing import Callable, Dict, Iterator, List, Optional, Set, Union
 
+from sqlalchemy.orm import Query, Session
+
 from cg.constants import FlowCellStatus, Pipeline
 from cg.constants.constants import PrepCategory, SampleType
 from cg.constants.indexes import ListIndexes
 from cg.exc import CaseNotFoundError, CgError
 from cg.store.api.base import BaseHandler
-from cg.store.filters.status_analysis_filters import AnalysisFilter, apply_analysis_filter
+from cg.store.filters.status_analysis_filters import (
+    AnalysisFilter,
+    apply_analysis_filter,
+)
 from cg.store.filters.status_case_filters import CaseFilter, apply_case_filter
-from cg.store.filters.status_case_sample_filters import CaseSampleFilter, apply_case_sample_filter
-from cg.store.filters.status_customer_filters import CustomerFilter, apply_customer_filter
-from cg.store.filters.status_flow_cell_filters import FlowCellFilter, apply_flow_cell_filter
+from cg.store.filters.status_case_sample_filters import (
```

Comment:
I am wondering if you have the same configuration of Isort/Black as me. This line is shorter than 100 characters and for me, it does not do any linting. I have Isort 5.12.0 and Black 23.1.0

Reply:
You should sync with the github action and pre-commit both running at 23.3.0